h2o-r/h2o-package/R/permutation_varimp.R

```diff
@@ -0,0 +1,23 @@
+#'
+#' Calculate Permutation Feature Importance 
+#'
+
+h2o.permutation_varimp <- function(model, frame){
+    if (is.H2OFrame(validation_frame))
+         tryCatch(permutation_varim_table <- .newExpr('Perm_Feature_importance', model, validation_frame), 
```

Comment:
Cant work. Be sure, every test locally passed before saying this is done.

```suggestion
         tryCatch(permutation_varim_table <- .newExpr('PermutationVarImp', model, validation_frame), 
```